x-pack/plugin/src/test/resources/rest-api-spec/api/autoscaling.put_autoscaling_policy.json

```
@@ -1,7 +1,8 @@
 {
   "autoscaling.put_autoscaling_policy":{
     "documentation":{
-      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/autoscaling-put-autoscaling-policy.html"
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/autoscaling-put-autoscaling-policy.html",
+      "description": "Create a new autoscaling policy."
```

nit for consistency:

```suggestion
      "description": "Creates a new autoscaling policy."
```